storage/aptosdb/src/utils/iterators.rs

```diff
@@ -149,14 +171,19 @@ impl<'a> PrefixedStateValueIterator<'a> {
                 }
 
                 if version > self.desired_version {
-                    self.inner
+                    self.kv_iter
+                        .as_mut()
+                        .unwrap()
```

Comment:
I feel it's more elegant to pass `kv_iter` in

Reply:
Made a variable.